src/features/ModelPlan/ModelToOperations/Home/index.tsx

```diff
@@ -35,7 +34,7 @@ const MTOHome = () => {
 
   const { modelName } = useContext(ModelInfoContext);
 
-  const { data, loading } = useGetModelToOperationsMatrixQuery({
+  const { data, loading, error, refetch } = useGetModelToOperationsMatrixQuery({
     variables: {
       id: modelID
     }
```

Comment:
the `useGetModelToOperationsMatrixQuery` is also used in `<MTOTable />` component.  So, I thought to pass down the data needed, instead of calling the gql again. 

Also, this allowed me to pass refetch down to the sibling component `<MTOTableActions />`